subprojects/packagefiles/speexdsp/meson.build

```diff
@@ -0,0 +1,113 @@
+project(
+  'speexdsp',
+  'c',
+  version : '1.2.0',
+  license : 'bsd-like',
```

Comment:
This seems to be, specifically, the `BSD-3-Clause` license, FWIW.

Reply:
Fixed; thanks.